src/metrics/mod.rs

```diff
@@ -589,7 +589,10 @@ request!(SORTED_SET_SCORE, "sorted_set_score");
  * PUBSUB
  */
 
-request!(PUBSUB_PUBLISH, "publisher/publish");
+counter!(PUBSUB_PUBLISH, "publisher/publish/total");
```

Comment:
This change isn't needed. the request macro expands to the 4 counters you have here.

Reply:
But using `request!` doesn't work. The snapshot.counter_rate(REQUEST_COUNTER) returns 0. Here is the output when using request!(PUBSUB_PUBLISH):
2023-10-18T00:06:25.662+00:00 Publishers: Current: 1
2023-10-18T00:06:25.662+00:00 Publisher Publish: Success: NaN % Timeout: NaN %
2023-10-18T00:06:25.662+00:00 Publisher Publish Rate (/s): Ok: 0.00 Error: 0.00 Timeout: 0.00
2023-10-18T00:06:25.662+00:00 Subscribers: Current: 1
2023-10-18T00:06:25.662+00:00 Subscriber Receive: Success: 100.00 % Corrupted: 0.00 %
2023-10-18T00:06:25.662+00:00 Subscriber Receive Rate (/s): Ok: 99.65 Error: 0.00 Corrupt: 0.00 Invalid: 0.00
2023-10-18T00:06:25.662+00:00 Pubsub Publish Latency (us): p25: 5341 p50: 5373 p75: 5373 p90: 10944 p99: 19136 p999: 19136 p9999: 19136
2023-10-18T00:06:25.662+00:00 Pubsub End-to-End Latency (us): p25: 5373 p50: 5406 p75: 5406 p90: 10944 p99: 19136 p999: 19136 p9999: 19136

Reply:
>>That’s because you’re using the wrong counter name. Should be PUBSUB_PUBLISH_COUNTER like it was before?

I borrowed the code from momento.rs that does PUBSUB_PUBLISH.increment()
If I am doing PUBSUB_PUBLISH_COUNTER.increment(), the compiler reports this error:
```error[E0599]: no method named `increment` found for reference `&'static str` in the current scope
   --> src/pubsub/kafka.rs:148:30
    |
148 |     PUBSUB_SUBSCRIBE_COUNTER.increment();
    |                              ^^^^^^^^^ method not found in `&str````